draft-ietf-oauth-transaction-tokens.md

```diff
@@ -553,6 +570,13 @@ This specification registers the following claims defined in Section {{txn-token
 * Change Controller: IESG
 * Specification Document: Section {{txn-token-header}} of this specificaiton
 
+## OAuth URI Subregistry Contents
+
+* URN: urn:ietf:params:oauth:token-type:self_signed
+* Common Name: Token type for Self-signed JWT in lieu of an access token
```

Comment:
Instead of "in lieu" could you please use a more well-known English phrase?

Reply:
Removed the additional phrase. The sentence makes sense without it.